site/docs/v1/tech/themes/kickstart-custom-theme.adoc

```diff
@@ -0,0 +1,400 @@
+
+- In the `volumes:` section of the FusionAuth service, add `- ./kickstart:/usr/local/fusionauth/kickstart`.
+
+- Modify `.env` and add the Kickstart configuration variable: `FUSIONAUTH_APP_KICKSTART_FILE=/usr/local/fusionauth/kickstart/kickstart.json`. This path should be what the Docker container expects, not the path on the host.
+
+- Configure `docker-compose.yml` to pass the environment variable set by `.env` to the container. Do this by adding `FUSIONAUTH_APP_KICKSTART_FILE: ${FUSIONAUTH_APP_KICKSTART_FILE}` to the `environment` section of the FusionAuth service.
+
+Now download the 5-minute guide files.
+
+```bash
+git clone https://github.com/FusionAuth/fusionauth-example-5-minute-guide \
+&& cd fusionauth-example-5-minute-guide
+```
+
+This folder contains a file called `.env.sample`:
+
+```env
+CLIENT_ID=CHANGEME
+CLIENT_SECRET=CHANGEME
+BASE_URL=http://localhost:9011
+```
+
+Change the `CLIENT_ID` and `CLIENT_SECRET` so that they match the `applicationId` and `clientSecret` variables from your `kickstart.json` file. Then save the file as `.env`
+
+```env
+CLIENT_ID=404e516b-06b8-49da-9c68-c1cd1928c81d
+CLIENT_SECRET=RBLhJrfRsa0-YxVPrn_aZfzIGccWyncdvHvDNTy-Hrs
+BASE_URL=http://localhost:9011
+```
+
+== Running Kickstart
+
+Once you have completed the steps above, you should have a folder that is structured as follows. We call this folder `Kickstart_Theme`, but you can call it whatever you like.
+
+```
++ Kickstart_Theme
+|
++-- docker-compose.yml
+|
++-- fusionauth-example-5-minute-guide
+|
++--+ kickstart
+   |
+   +-- kickstart.json
+   |
+   +-- darkTheme.css
+   |
+   +--+ json
+      |
+      +-- createTheme.json
+      |
+      +-- updateTheme.json
+      |
+      +-- createUser.json
+      |
+      +-- registerUser.json
+      |
+      +-- createApplication.json
+```
+
+To launch the FusionAuth instance, navigate to the `Kickstart_Theme` folder and run the Docker Compose file.
+
+```bash
+docker-compose up
+```
+
+Once the execution has finished, the newly created FusionAuth instance will be accessible at `http://localhost:9011`.
+
+Login to the FusionAuth instance. The username and password are configured in `kickstart/json/createUser.json`. You can set them to be anything you like, but for this tutorial, they are defined as follows:
+
+```json
+"email": "test_user@email.com",
+"password": "1234567890"
+```
+
+Enter these credentials into the login screen to be taken to the admin dashboard.
+
+You can look at [breadcrumb]#Applications#, [breadcrumb]#Users#, and [breadcrumb]#Customizations -> Themes# to verify that all of the settings have been configured correctly.
+
+Before we can see our custom theme in action, we need to apply it to the application.
+
+== Applying the Theme
+
+To apply our custom theme, we need to bind it to the default tenant. Since the default tenant is created automatically with a random UUID every time, programmatically modifying it is outside the scope of this tutorial.
```

Comment:
Actually, you can specify the UUID: https://fusionauth.io/docs/v1/tech/installation-guide/kickstart#modify-the-default-tenant-id

If you set it as a variable, then you can automatically PATCH it to apply the theme. Can you please do that and modify this section so there's no manual process involved?

Unless there's some value in the manual process that I'm not seeing.